app/src/main/res/values/strings.xml

```diff
@@ -49,6 +49,14 @@
     <string name="sender_and_message">Sender and message</string>
     <string name="sender_only">Sender only</string>
     <string name="enable_delivery_reports">Enable delivery reports</string>
+    <string name="mms_file_size_limit">MMS image size limit</string>
```

Comment:
change the string to "Resize sent MMS images", then change "1000KB" to "1MB", 2000 to 2 and set 1MB as default